cmd/print.go

```diff
@@ -23,7 +25,17 @@ func NewPrintCmd() *cobra.Command {
 				return err
 			}
 
-			printEnv(values, cmd.Flag("output").Value.String())
+			keys := r.LoadedVars
```

Comment:
This pattern's pretty hacky, planning to refactor the resolver type soon. All internal, so not a problem aside from modest clunkiness.